build/builtin_actors_gen.go

```diff
@@ -140,9 +140,32 @@ var EmbeddedBuiltinActorsMetadata = []*BuiltinActorsMetadata{{
 		"system":           cid.MustParse("bafk2bzaceacjmlxrvydlud77ilpzbscez46yedx6zjsj6olxsdeuv6d4x4cwe"),
 		"verifiedregistry": cid.MustParse("bafk2bzacebs5muoq7ft2wgqojhjio7a4vltbyprqkmlr43ojlzbil4nwvj3jg"),
 	},
+}, {
+	Network: "butterflynet",
```

Comment:
`BundleGitTag` is missing from each of the `Version: 14` entries and is in the `15`s, is that the way it's supposed to be?

It would be nice if the old ones were unmodified so the diff made it obvious that the new ones come last, but as it is, it makes it look like the new ones are second last